local_build.sh

```diff
@@ -10,21 +10,27 @@ RET=0
 
 mkdir -p $OUTDIR
 
+DOCKER=docker
+if ! type -P ${DOCKER} 2>/dev/null;
+then
+    DOCKER=podman
+fi
+
```

Comment:
Oh nice. I'm still a newbie in all this. I never thought we could use podman directly :+1:

Reply:
I've got `alias docker=podman` in my bashrc :eyes: